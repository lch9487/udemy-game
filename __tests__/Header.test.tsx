import * as React from 'react';
import { mount } from 'enzyme';
import Header from '../components/Header';

describe('Header test', () => {
  it('should render component correctly', () => {
    const wrapper = mount(<Header />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
