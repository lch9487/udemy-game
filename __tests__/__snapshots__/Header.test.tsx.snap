// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test should render component correctly 1`] = `
"<Header>
  <div>
    <Link href=\\"/\\">
      <a style={{...}} onClick={[Function: onClick]} href=\\"/\\">
        Home
      </a>
    </Link>
    <Link href=\\"/about\\">
      <a style={{...}} onClick={[Function: onClick]} href=\\"/about\\">
        About
      </a>
    </Link>
  </div>
</Header>"
`;
